src/script/components/AvailabilityState.tsx

```diff
@@ -25,25 +25,24 @@ import cx from 'classnames';
 import {Availability} from '@wireapp/protocol-messaging';
 
 import {selfIndicator} from 'Components/ParticipantItemContent/ParticipantItem.styles';
-import {VerificationBadges} from 'src/script/components/VerificationBadges';
+import {useKoSubscribableChildren} from 'Util/ComponentUtil';
 import {CSS_SQUARE} from 'Util/CSSMixin';
 import {KEY} from 'Util/KeyboardUtil';
 
 import {Icon} from './Icon';
 
+import {User} from '../entity/User';
+
 interface AvailabilityStateProps {
-  availability: Availability.Type;
+  user: User;
   className?: string;
   dataUieName: string;
-  label: string;
   selfString?: string;
   title?: string;
   onClick?: (event: React.MouseEvent | React.KeyboardEvent) => void;
   showArrow?: boolean;
   theme?: boolean;
-  showBadges?: boolean;
-  isProteusVerified?: boolean;
-  isMLSVerified?: boolean;
+  children?: React.ReactNode;
```

Comment:
It's not the responsibility of the caller to give the badges (if need be). 
Now this component doesn't have any awareness of the existence of badges